analytics.proto

```diff
@@ -51,6 +51,15 @@ enum AnalyticsEventType {
   RECORDING_ENDED = 9;
 }
 
+enum SdkType {
```

Comment:
We already have ClientInfo.SDK, can we use that instead of creating a new one?

```
message ClientInfo {
  enum SDK {
     UNKNOWN = 0;
     JS = 1;
     IOS = 2;
     ANDROID = 3;
     FLUTTER = 4;
     GO = 5;
  }
```